src/main/native/include/tcpsockets/TCPAcceptor.h

```diff
@@ -6,7 +6,7 @@
 
    ------------------------------------------
 
-   Copyright � 2013 [Vic Hargrave - http://vichargrave.com]
+   Copyright � 2013 [Vic Hargrave - http://vichargrave.com]
```

Comment:
The copyright symbol got mangled. UTF problems? We could either find the right symbol or replace it with "(c)".